### DISCUSSION TITLE
UI issue when clicking on the "reaction" button

### DISCUSSION THREAD

**Discussion by endel** [community/community#35] [Discussions] [answered]
When I click for the first time on the "reaction" icon, the options are outside the page. After I click on any reaction, and try to open the reactions box again, it shows up properly.

<img width="261" alt="Screen Shot 2020-10-26 at 20 02 34" src="https://user-images.githubusercontent.com/130494/97237648-514e9500-17c6-11eb-9269-fd90754e2196.png">

**Comment by becca** [comment, marked as answer]
Thank you for opening this Discussion. This is more of a bug than feedback but it is duly noted and a problem we're working to solve this month.
